components/docs-chef-io/content/automate/chef_infra_server.md

```diff
@@ -0,0 +1,110 @@
+
+To add the created Chef Server to the Infrastructure, click on _Add Chef Server_ button. Clicking on the Add Chef Server button opens a popup menu as shown below:
+
+![Add Chef Server Popup Menu](/images/automate/add-chef-server-popup-menu.png)
+
+Mention the correct Name, FQDN, and IP Address of the system. Click _Add Chef Server_. The desired server gets added to the list of  Chef Infra Server.
+
+**Note:** If the mentioned FQDN and IP Address is incorrect, the box displays an error as shown below:
+
+![Add Chef Server Popup Menu](/images/automate/add-chef-server-popup-menu-with-error.png)
+
+Click on the listed Servers to view the details of the specific _Chef Server_.
+
+## Connect to the Chef Organization
+
+Selecting a specific added Chef Server opens the list of organizations in the Server. Refer to the image below:
+
+![Chef Infra Server Organization](/images/automate/chef-server-organization.png)
+
+The Organization in the Chef server listed are the ones which are already created in that specific server. To create a new Organization in the Server click [here](https://docs.chef.io/automate/infra_server/#set-up-the-chef-infra-server).
+
+To add the previously created organization to the Infrastructure, click **Add Chef Organization** button. Clicking on the button opens a popup menu as shown below:
+
+![Add Chef Organization Popup Menu](/images/automate/add-chef-organization-popup-menu.png)
+
+Add the Name, Projects, Admin User, and Admin Key. Always use the exact details that were provided at the time of setting up _Knife_. Copy the contents of the `~/.chef/USER.pem` file and paste it into the _Admin Key_ field. Once done click **Add Chef Organization**. The desired organization will be added to the Chef Server.
+
+Click on any Chef Organization to view the _Objects_.
+
+## Access Chef Infra Server Objects
+
+The Chef Infra Server Objects can be managed from the Chef Automate. In Automate Infra views, following objects can be managed:
+
+- Cookbooks
+- Roles
+- Environments
+- Data Bags
+- Clients
+
+Refer to the image below:
+
+![Chef Infra Server Objects](/images/automate/chef-infra-server-objects.png)
+
+### Cookbooks
+
+A [cookbook](https://docs.chef.io/cookbooks/) is the fundamental unit of configuration and policy distribution. Include recipes and other files, components, or directories.
+
+### Roles
+
+[Roles](https://docs.chef.io/roles/) let you define patterns and processes that exist across nodes in an organization as belonging to a single job function. Each role consists of zero (or more) attributes and a run-list.
+
+### Environments
+
+An [environment](https://docs.chef.io/environments/) can be used to map an organization’s real-life workflow to what can be configured and managed when using Chef Infra. This mapping is accomplished by setting attributes and pinning cookbooks at the environment level.
```

Comment:
Change `organization’s`  to `organization's"

I think you may have smart quotes enabled, if so, change them to straight quotes. This will save you a lot of frustration. https://support.microsoft.com/en-us/office/smart-quotes-in-word-702fc92e-b723-4e3d-b2cc-71dedaf2f343